tsconfig.json

```diff
@@ -12,6 +12,8 @@
     "strictNullChecks": true,
     "esModuleInterop": true,
     "types": ["@wdio/types", "node"],
-    "target": "es2019"
+    "target": "es2019",
+    "outDir": "./output",
```

Comment:
we usually name this folder dist instead of output.
if you do change it, also update the gitignore file

Reply:
I couldn't get dist to work, I could only get output to work. 
There must be something messing with the creation of dist folder somewhere